src/core/Object3D.js

```diff
@@ -696,6 +697,8 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 				generator: 'Object3D.toJSON'
 			};
 
+			this.updateMatrixWorld( true );
```

Comment:
This is necessary to ensure a correct bone hierarchy. Without this line, `Object3D.toJSON()` might produce identity matrices if `position`, `quaternion` and `scale` are not composed into a matrix.